src/docs/SUPPORTED_MODELS.md

```diff
@@ -167,14 +167,23 @@ The pipeline can work with other similar topologies produced by `optimum-intel`
       <th>Example HuggingFace Models</th>
     </tr>
     <tr>
-      <td><code>MiniCPM-V-2_6</code></td>
       <td>MiniCPMV</td>
+      <td><code>MiniCPM-V-2_6</code></td>
       <td>
         <ul>
           <li><a href="https://huggingface.co/openbmb/MiniCPM-V-2_6"><code>openbmb/MiniCPM-V-2_6</code></a></li>
         </ul>
       </td>
     </tr>
+    <tr>
+      <td>LLaVA</td>
```

Comment:
Please, sort it alphabetically by placing above MiniCPM

Reply:
Fixed